src/decoder_vt.c

```
@@ -309,6 +311,19 @@ static uint32_t pix_fmt_ff2vt(const char *fmt_str)
     return -1;
 }
 
+static OSType get_cv_pix_fmt(enum AVPixelFormat pix_fmt)
+{
+    const AVPixFmtDescriptor *desc = av_pix_fmt_desc_get(pix_fmt);
+    if (!desc)
+        return kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange;
```

Maybe return an error in this case? Either by using a dedicated `OSType` if it exists, or checking the descriptor in the caller side and passing down the `desc` to the function if everything went well.

Because if this case happen, something looks very fishy.